packages/react/src/primitives/types/menu.ts

```diff
@@ -38,6 +38,12 @@ export interface MenuProps extends FlexProps {
    * Note: only applies if `trigger` prop is null
    */
   triggerClassName?: string;
+
+  /**
+   * Whether or not the menu should use a React Portal
+   * @default false
+   */
+  portalled?: boolean;
```

Comment:
Should we follow our boolean variable naming convention here?
```suggestion
  isPortalled?: boolean;
```

Reply:
Curious what the use case is for customers not having the menu in a React portal? Mainly for styling?